src/shutdown.cc

```diff
@@ -246,6 +246,24 @@ class subproc_buffer : private cpbuffer<subproc_bufsize>
     }
 };
 
+static bool
```

Comment:
match style used elsewhere - the return type should be on the same line as the function name.